Applications/Games/Space Engineers/Steam/script.js

```diff
@@ -0,0 +1,30 @@
+include(["engines", "wine", "quick_script", "steam_script"]);
+include(["engines", "wine", "verbs", "dotnet461"]);
+include(["engines", "wine", "verbs", "vcrun2015"]);
+include(["engines", "wine", "verbs", "dxvk"]);
+include(["engines", "wine", "verbs", "xact"]);
+
+var installerImplementation = {
+    run: function () {
+        new SteamScript()
+            .name("Space Engineers")
+            .editor("Keen Software House")
+            .author("Zemogiter")
+            .appId(244850)
+            .wineVersion("3.20")
+            .wineDistribution("staging")
+            .wineArchitecture("amd64")
+            .preInstall(function (wine, wizard) {
+                wizard.message(tr("This script will not work on Mac OS until MoltenVK is merged into Wine."));
```

Comment:
Can't we abort the script if we are on Mac OS? see `Bean("operatingSystemFetcher")`